sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -184,27 +192,14 @@ export default {
       sortField = StudentApplicationFields.Status,
       sortOrder = DataTableSortOrder.ASC,
     ) => {
-      switch (clientType.value) {
-        case ClientIdType.Student:
-          applicationAndCount.value =
-            await StudentService.shared.getAllStudentApplications(
-              page,
-              pageCount,
-              sortField,
-              sortOrder,
-            );
-          break;
-        case ClientIdType.AEST:
-          applicationAndCount.value =
-            await ApplicationService.shared.getAllApplicationAndCount(
-              props.studentId,
-              page,
-              pageCount,
-              sortField,
-              sortOrder,
-            );
-          break;
-      }
+      applicationsAndCount.value =
+        await ApplicationService.shared.getStudentApplicationSummary(
```

Comment:
This service method can use the PaginationOptions interface instead of 5 arguments.

Reply:
It is the Web project and I will not refactor it as part of this PR.